### ISSUE TITLE
new bookkeeper-cli doesn't work in binary distribution

### ISSUE THREAD

**Issue by sijie** [apache/bookkeeper#1356]
**BUG REPORT**

1. Please describe the issue you observed:

- What did you do?

- run standalone in docker
- docker exec -it <container id> bash
- run bin/bookkeeper-cli

- What did you expect to see?

bookkeeper-cli should print help usage

- What did you see instead?

```
# bin/bookkeeper-cli
Couldn't find bookkeeper jar.
Do you want me to run `mvn package -DskiptTests` for you ?
```

**Comment by sijie** [comment]
this needs to be fix before we claim [BP-27](https://github.com/apache/bookkeeper/issues/1000) #1000 as done.

**Comment by sijie** [comment]
this is fixed by b9b46172c4f7ac1aa2a42ff8b0fceb24ef03ed68

**Comment by sijie** [comment]
This is fixed by #1478
